active-mq-artemis/scripts/broker-00.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think this should stay out of the module folder (a global scripts folder is fine), since it’s only needed for local dev and tests.